sale_by_packaging/tests/test_sale_only_by_packaging.py

```diff
@@ -121,8 +66,11 @@ def test_create_auto_fill_packaging(self):
                 "product_uom_qty": self.packaging.qty * 2,
             }
         )
-        self.assertFalse(order_line_1.product_packaging)
-        self.assertFalse(order_line_1.product_packaging_qty)
+        # Why should it be false if the quantity fits ?
```

Comment:
@TDu is this something still needed?

Reply:
I kept it commented because, I did not understand the assertion.
For me it can be removed.

Reply:
removed